Recipes/Along-slope-velocities.ipynb

```diff
@@ -23,6 +23,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cat_filtered = catalog.search(name=EXPERIMENT)
</pre><p>This line can go directly to the experiment by key instead:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">cat_filtered = catalog[EXPERIMENT]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>